packages/algob/src/types.ts

```diff
@@ -545,6 +545,10 @@ export interface StrMap {
   [key: string]: string
 }
 
+export interface ScParamsMap {
+  [key: string]: string | bigint
```

Comment:
Shouldn't we include numbers as well? We recommend `bigint`, but i think we should have `number` as well. Right?

Reply:
https://github.com/scale-it/algorand-builder/issues/243#issuecomment-781930008

Reply:
I am not sure if it means to remove the support for `number` and use only `bigint`. Ignore if i misunderstood.